radio/src/pulses/afhds3_config.h

```diff
@@ -129,7 +129,8 @@ PACK(struct sDATA_ConfigV1 {
 
 PACK(struct sDATA_Others {
   uint8_t buffer[sizeof(sDATA_ConfigV1)];
-  uint8_t ExternalBusType;  // eEB_BusType
+  uint8_t ExternalBusType;  // eEB_BusType IBUS1:0;IBUS2:1(Not supported yet);SBUS:2
+  uint8_t iBusType;  //IBUS-OUT:0; IBUS-IN:1
```

Comment:
Let's use a clean `enum` for these 2.

Reply:
with
```suggestion
  enum IBusType: uint8_t {IBusOut = 0, IBusIn = 1}:
```
you can retain the same size of the data struct

Reply:
I have made some changes.  See if this is good to you all.